source/postprocess/visualization/boundary_velocity_residual.cc

```diff
@@ -0,0 +1,243 @@
+                  }
+              }
+          }
+      }
+
+
+      template <int dim>
+      void
+      BoundaryVelocityResidual<dim>::declare_parameters (ParameterHandler  &prm)
+      {
+        prm.enter_subsection("Postprocess");
+        {
+          prm.enter_subsection("Visualization");
+          {
+            prm.enter_subsection("Surface properties");
+            {
+              prm.declare_entry ("Data directory",
+                                 "$ASPECT_SOURCE_DIR/data/boundary-velocity/gplates/",
+                                 Patterns::DirectoryName (),
+                                 "The name of a directory that contains the GPlates model or the "
+                                 "ascii data. This path may either be absolute (if starting with a "
+                                 "`/') or relative to the current directory. The path may also "
+                                 "include the special text `$ASPECT_SOURCE_DIR' which will be "
+                                 "interpreted as the path in which the ASPECT source files were "
+                                 "located when ASPECT was compiled. This interpretation allows, "
+                                 "for example, to reference files located in the `data/' subdirectory "
+                                 "of ASPECT.");
+              prm.declare_entry ("Data file name", "current_day.gpml",
+                                 Patterns::Anything (),
+                                 "The file name of the input velocity as a GPlates model or an ascii data. "
+                                 "For the GPlates model, provide file in the same format as described "
+                                 "in the gplates boundary velocity plugin. "
+                                 "For the ascii data, provide file in the same format as described in "
+                                 " 'ascii data' initial composition plugin." );
+              prm.declare_entry ("Scale factor", "1.",
+                                 Patterns::Double (),
+                                 "Scalar factor, which is applied to the model data. "
+                                 "You might want to use this to scale the input to a "
+                                 "reference model. Another way to use this factor is to "
+                                 "convert units of the input files. For instance, if you "
+                                 "provide velocities in cm/yr set this factor to 0.01.");
+              prm.declare_entry ("Use spherical unit vectors", "false",
+                                 Patterns::Bool (),
+                                 "Specify velocity as r, phi, and theta components "
+                                 "instead of x, y, and z. Positive velocities point up, east, "
+                                 "and north (in 3D) or out and clockwise (in 2D). "
+                                 "This setting only makes sense for spherical geometries.");
```

Comment:
Add: "GPlates data is always interpreted to be in east and north directions and is not affected by this parameter."